.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,53 @@
+
+## Fixes 
+<!-- For e.g Fixes: https://issues.redhat.com/browse/HAC-XXX -->
+
+
+## Description
+<!-- Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change. -->
+
+
+## Type of change
+<!-- Please delete options that are not relevant. -->
+
+- [ ] Feature
+- [ ] Bugfix
+- [ ] Code style update (formatting, renaming)
+- [ ] Refactoring (no functional changes, no api changes)
+- [ ] Build related changes
+- [ ] Documentation content changes
+- [ ] Other (please describe):
+
+## Screen shots / Gifs for design review 
+<!-- If change affects UI in any way, tag relevant UX people and add screenshots/gifs  -->
```

Comment:
If we want users to tag the correct groups, we should have groups and list them here.

Reply:
Not sure if we have proper UX groups at this point. The one we used for devconsole - `openshift/team-devconsole-ux` doesn't seem to be updated with new members. We can update this later.